devpi_builder/requirements.py

```diff
@@ -61,7 +61,7 @@ def matched_by_list(package, version, requirements):
     version = pkg_resources.safe_version('{}'.format(version))
     package = pkg_resources.safe_name(package)
     matches = (
-        package == requirement.project_name and version in requirement
+        package.lower() == requirement.key and version in requirement
```

Comment:
Just wondering. Will key always be in lower case or should that also be explicitly lower-cased?

Reply:
According to https://setuptools.readthedocs.io/en/latest/pkg_resources.html:

```
key
dist.key is short for dist.project_name.lower(). It’s used for case-insensitive comparison and indexing of distributions by project name.
```